src/main/java/com/google/sps/servlets/NewUserServlet.java

```diff
@@ -21,67 +21,49 @@
 import com.google.sps.user.User;
 import com.google.sps.user.repository.UserRepository;
 import com.google.sps.user.repository.UserRepositoryFactory;
+import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
 
 /**
- * This servlet handles users' data.
+ * This servlet checks if the currently logged in user exists in the database and saves a new user
+ * in the database.
  */
-@WebServlet("/user-data")
-public class UserServlet extends HttpServlet {
-  public static final String NAME_INPUT = "name";
-  public static final String PUBLIC_PORTFOLIO_INPUT = "publicPortfolio";
-  public static final String PUBLIC_PORTFOLIO_INPUT_PUBLIC_VALUE = "public";
-  public static final String SELF_INTRODUCTION_INPUT = "selfIntroduction";
-  public static final String IMG_URL_INPUT = "imgUrl";
-
+@WebServlet("/new-user-servlet")
+public class NewUserServlet extends HttpServlet {
```

Comment:
Maybe UserCreationServlet is better name

Maybe also explain that this is because we use Google account for login. The first time user logins our site, we create create and store the user in the our data

Reply:
Changed name and completed javadoc.